en/admin/config.rst

```diff
@@ -3110,6 +3112,10 @@ Etc
 
     An environment variable, if exists, has priority over **cubrid.conf**. The default value is **cubrid.env**.
 
+**NET_BUF_SIZE**
+
+    **NET_BUF_SIZE** is a parameter to determine the network buffer size used by **CAS** to transmit query results to the client. The query result sets will be buffered into the **CAS** network buffer, and it will be sent to the client when the buffer is full. **NET_BUF_SIZE** is the size of the packet that CAS transmits to the client, and it may affect transmission efficiency.
```

Comment:
```suggestion
    **NET_BUF_SIZE** is a parameter used to determine the network buffer size used by **CAS** to transmit query results to the client. The query result sets will be buffered into the **CAS** network buffer, and are sent to the client when the buffer is full. **NET_BUF_SIZE** is the size of the packet that CAS transmits to the client, and it may affect transmission efficiency.
```

Reply:
schemainfo에 대한 결과셋을 fetch할때도 사용하여서 query results라고 국한 것보다 results로 변경하는 것은 어떨까요 ?

Reply:
@mhoh3963 ,
Comment 감사합니다. 더 정확한 표현인 것 같습니다.

@junsklee 
문장이 훨씬 부드러워 졌습니다. 빠른 review 감사드립니다.